packages/angular/src/generators/stories/__snapshots__/stories-app.spec.ts.snap

```diff
@@ -175,6 +175,7 @@ Object {
     101,
     110,
     116,
+    44,
```

Comment:
Could you update the test cases for this snapshot so we match the actual `string` instead of a `Buffer`? This is an existing issue, but let's take the chance to correct it.